velox/core/Config.cpp

```diff
@@ -32,6 +32,14 @@ bool MemConfig::isValueExists(const std::string& key) const {
   return values_.find(key) != values_.end();
 }
 
+void MemConfig::initialize() {
+  if (auto it = values_.find(QueryConfig::kSessionTimezone);
```

Comment:
can you use get method here? https://github.com/facebookincubator/velox/pull/9714/files#diff-9a775066f1d8ed599d62ea1e5adcaa43f7c7d093aa74cd2e98b571195fe38095R22